views/index.ejs

```diff
@@ -61,7 +61,8 @@
       <%- include navbar %>
       <%- include overlay %>
       <%- include title %>
-      <div class="g mnone">
+      //SplitJS
+      <div id="split-area" class="g mnone" ng-controller="diSplitJs">
         <%- include editor-headers %>
```

Comment:
I would move the editors-header outside this div. Eventually the headers will probably need to be included with their corresponding section, but for now they're a third element and we need there to be two